generative/maisi/scripts/find_masks.py

```diff
@@ -0,0 +1,122 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import json
+import os
+import zipfile
+
+
+def convert_body_region(
+    body_region: list[int],
+):
+    body_region_indices = []
+
+    for _k in range(len(body_region)):
+        region = body_region[_k].lower()
+
+        idx = None
+        if "head" in region:
+            idx = 0
+        elif "chest" in region or "thorax" in region or "chest/thorax" in region:
+            idx = 1
+        elif "abdomen" in region:
+            idx = 2
+        elif "pelvis" in region or "lower" in region or "pelvis/lower" in region:
+            idx = 3
+        else:
+            raise ValueError("Input region information is incorrect.")
+
+        body_region_indices.append(idx)
+
+    return body_region_indices
+
+
+def find_masks(
+    body_region: str | list[str],
+    anatomy_list: int | list[int],
+    spacing: list[float],
+    output_size: list[int],
+    check_spacing_and_output_size: bool = False,
+    database_filepath: str = "./database.json",
+    mask_foldername: str = "./masks",
+):
+    if type(body_region) is str:
+        body_region = [body_region]
+
+    body_region = convert_body_region(body_region)
+
+    if type(anatomy_list) == int:
+        anatomy_list = [anatomy_list]
+
+    if not os.path.isfile(database_filepath):
+        raise ValueError(f"Please download {database_filepath}.")
```

Comment:
It would be better to mention what this file is for, or where it can be downloaded.